changelog.org

```diff
@@ -681,463 +681,453 @@ Changelog]]
   [[https://github.com/anoma/juvix/pull/1320][#1320]]
   ([[https://github.com/mariari][mariari]])
 
-** [[https://github.com/anoma/minijuvix/tree/v0.2.1][v0.2.1]] (2022-07-12)
 
-[[https://github.com/anoma/minijuvix/compare/v0.2.0...v0.2.1][Full
-Changelog]]
+** New name: Juvix
+
+Since version 0.2.2, the project has been renamed from "Mini Juvix" to
+"Juvix". The new name reflects the fact that the project is no longer
+just a compiler for a subset of Juvix, but a full implementation of the
+language. Affected by this change are:
+
+- Github repository moved from the helium organization to the anoma
```

Comment:
Wasn't this supposed to be heliax instead of helium?

Reply:
Sure. Funny.